scalariform/src/test/scala/scalariform/lexer/ScalaLexerTest.scala

```diff
@@ -10,7 +10,7 @@ class ScalaLexerTest extends FlatSpec with Matchers {
                                             scalaVersion: ScalaVersion = ScalaVersions.DEFAULT): TestString =
     new TestString(s, forgiveErrors, scalaVersion)
 
-  "" producesTokens ()
+  "" producesTokens (List.empty[TokenType]: _*)
```

Comment:
an alternative would be to avoid infix syntax and just write `"".producesTokens()`

Reply:
didn't look so nice with the rest of the file. :)